k8s/k8s.go

```diff
@@ -70,6 +70,15 @@ type Ops interface {
 	SnapshotOps
 	SecretOps
 	ConfigMapOps
+	EventsOps
```

Comment:
nit: EventOps to be coherent with other interfaces

Reply:
done